frontend/base/src/test/java/com/virtuslab/gitmachete/resourcebundles/junit/GitMacheteBundlePropertiesTestSuite.java

```diff
@@ -28,4 +29,20 @@ public void htmlProperties_should_have_correct_syntax() throws IOException {
           assertEquals("HTML property key should have a .HTML suffix", ".HTML", key.substring(key.length() - 5));
         });
   }
+
+  @Test
+  public void properties_should_use_ellipsis_instead_of_three_dots() throws IOException {
+
+    Properties property = new Properties();
+    ClassLoader loader = Thread.currentThread().getContextClassLoader();
+    property.load(loader.getResourceAsStream(macheteBundleProperties));
+
+    property.entrySet().stream()
+        .filter(prop -> ((String) prop.getValue()).contains("..."))
+        .forEach(t -> {
+          String key = (String) t.getKey();
+          fail("Properties should use ellipsis (\\u2026) instead of three dots (key=${key})");
```

Comment:
While you're here... pls also make sure that error messages from `html_properties_should_have_correct_syntax` also report the faulty key (looks that they don't now)